samples/ibm-samples/ffdl-seldon/source/README.md

```diff
@@ -0,0 +1,8 @@
+# Source Code for IBM OSS pipeline demo using PyTorch.
+
+This dictretory contains the source code for the PyTorch Gender Classification demo using FfDL and Seldon on KubeFlow Pipeline.
+
+- [manifest.yml](manifest.yml): Manifest file for FfDL training.
+- [gender-classification.zip](gender-classification.zip): Packaged model definition file.
```

Comment:
Is there a way to avoid checking in the zip files, tar files, and binaries, and generate them instead when they are needed?

Reply:
Sure @vicaire. We will remove the zip and add instructions to zip